pkgs/misc/tmux-plugins/default.nix

```
@@ -818,4 +818,22 @@ in rec {
       sha256 = "hRvkBf+YrWycecnDixAsD4CAHg3KsioomfJ/nLl5Zgs=";
     };
   };
+
+  tmux-nova = mkTmuxPlugin rec {
+    pluginName = "tmux-nova";
```

It seems like nixpkgs looks for the file named after your package, i.e `tmux_nova.tmux`. 
You can specify a different file by setting:
```suggestion
    pluginName = "tmux-nova";
    rtpFilePath = "nova.tmux";
```